bsmetadata/metadata_processors.py

```diff
@@ -103,10 +103,18 @@ def process_global(self, metadata_attrs: Dict[str, Any]) -> Optional[str]:
         # Example: "foo.bar/Year 2021/" instead of "foo.bar/Year%202021/".
         return "".join([metadata_attrs["key"], self.cfg.metadata_key_value_sep, unquote_plus(metadata_attrs["value"])])
 
+class WebsiteDescriptionProcessor(MetadataProcessor):
+    """An example metadata processor for URLs."""
```

Comment:
A very small change suggestion :slightly_smiling_face: 
```suggestion
    """An example metadata processor for website descriptions."""
```